src/OrchardCore.Modules/OrchardCore.Layers/Services/DefaultLayersMethodProvider.cs

```diff
@@ -69,8 +71,19 @@ public DefaultLayersMethodProvider()
                         : string.Equals(requestPath, url, StringComparison.OrdinalIgnoreCase); ;
                 })
             };
+
+            _currentCulture = new GlobalMethod
+            {
+                Name = "currentCulture",
+                Method = serviceProvider => (Func<string, object>)(culture =>
+                {
+                    var currentCulture = CultureInfo.CurrentCulture.Name.ToLowerInvariant();
```

Comment:
You don't need `ToLowerInvariant` because the check is already culture invariant.